config/index.js

```diff
@@ -472,6 +472,51 @@ var conf = convict({
       default: true,
       env: 'SECURITY_HISTORY_ENABLED'
     }
+  },
+  signinUnblock: {
+    codeLength: {
+      doc: 'Number of base36 digits to make up an unblockCode',
+      default: 8,
+      env: 'SIGNIN_UNBLOCK_CODE_LENGTH'
+    },
+    codeLifetime: {
+      doc: 'How long an unblockCode should be valid for',
+      format: 'duration',
+      default: '1 hour',
+      env: 'SIGNIN_UNBLOCK_CODE_LIFETIME'
+    },
+    enabled: {
+      default: true
+    },
+    allowEmailRegex: {
+      doc: 'If feature enabled, allow sign-in unblock for email addresses matching this regex.',
+      format: Array,
+      default: [
+        '.+@mozilla\.com$'
+      ],
+      env: 'SIGNIN_UNBLOCK_ALLOW_EMAIL_REGEX'
+    },
+    sample_rate: {
```

Comment:
For the record, I've added a `sampleRate` over in #1470. The snake case should be terminated with extreme prejudice imho, I'm happy to do it in my PR or if you want to do it here that's cool as well.